src/playwright-test.ts

```diff
@@ -0,0 +1,401 @@
+import { once } from 'events';
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { Shell } from 'electron';
+import {
+  _electron,
+  ConsoleMessage,
+  Dialog,
+  ElectronApplication,
+  Page,
+} from 'playwright';
+
+import { getTempDir } from './helpers/helpers';
+import { NativefierOptions } from '../shared/src/options/model';
+
+const INJECT_DIR = path.join(__dirname, '..', 'app', 'inject');
+
+const log = console;
+
+function sleep(milliseconds: number): Promise<void> {
+  return new Promise((resolve) => {
+    setTimeout(resolve, milliseconds);
+  });
+}
+
+describe('Application launch', () => {
```

Comment:
General question: now that this automates several of the manual smoke tests, should you remove said now-irrelevant-because-automated smoke tests? I'd nuke them, no point in keeping including them in the smoke suite. The smoke suite should remain as last resort for tests we *cannot* test within playwright. Yea/nay?

Reply:
Sounds right to me. I'll trim those as well.